pkg/backend/identityapi/register_client.go

```diff
@@ -64,13 +64,15 @@ type RegisterEntityResponse struct {
 	ID   entity.ID  `json:"entityID"`
 	Key  entity.Key `json:"entityKey"`
 	Name string     `json:"entityName"`
+	Err  string     `json:"err"`
```

Comment:
I believe that what register client responde is `json:"error"`. Or at least that is what they documented

Reply:
As I got it, we can omit warnings in data, but log them in the warn log. If warning happened, we are still getting entityId.